src/components/multiview/MultiviewSyncBar.vue

```diff
@@ -353,10 +361,6 @@ export default {
                         cell.seekTo(expectedDuration);
                     }
                 });
-            // Wait for all videos to exit buffering state (3) to increment currentTs, if not paused
-            if (!states.includes(3) && !this.paused) {
```

Comment:
Note that `states` was never written to, so I have hoisted this up so that we update the players immediately and the `states` constant is deleted.